app/models/case_log.rb

```diff
@@ -1,4 +1,53 @@
 class CaseLogValidator < ActiveModel::Validator
+  # Methods to be used on save and continue need to be named 'validate_'
+  # followed by field name this is how the metaprogramming of the method
+  # name being call in the validate method works.
+  def validate_property_number_of_times_relet(record)
```

Comment:
This looks like it's still coming from merge? Don't think this file needs to change at all?